tests/platform_helper/domain/test_database_copy.py

```diff
@@ -480,11 +482,14 @@ def test_database_copy_account_id():
 
 def test_update_application_from_platform_config_if_application_not_specified(fs):
     fs.create_file(PLATFORM_CONFIG_FILE, contents=yaml.dump({"application": "test-app"}))
-    mocks = DataCopyMocks()
 
-    # TODO - remove, see other comments on patches
-    with patch("dbt_platform_helper.utils.validation.boto3.client"):
-        db_copy = DatabaseCopy(None, "test-db", **mocks.params())
+    config_validator = Mock()
+    config_validator.run_validations.return_value = None
+    config_provider = ConfigProvider(config_validator)
```

Comment:
If we allow config_validator = None, then this sort of thing would be easier too